src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/DSAKeyValue.cs

```diff
@@ -186,12 +186,12 @@ public override void LoadXml(XmlElement value)
                 Key.ImportParameters(new DSAParameters
                 {
                     P = (pNode != null) ? Convert.FromBase64String(pNode.InnerText) : null,
-                    Q = (pNode != null) ? Convert.FromBase64String(qNode.InnerText) : null,
+                    Q = (qNode != null) ? Convert.FromBase64String(qNode.InnerText) : null,
```

Comment:
This doesn't change much, since P and Q can only occour in combination or none of them. I thought it would be easier to read if I stay with the condition `q = (pNode != null) ...`